patches/lua-tree.patch

```diff
@@ -337,31 +338,49 @@ index c1a150a..3d6c639 100644
      return nil,
      "invalid value '" .. name ..
        "': the only accepted ascii characters are alphanumerics or ., -, _, and ~"
-@@ -216,22 +215,12 @@ end
- 
- 
- local function validate_certificate(cert)
+@@ -215,26 +214,6 @@
+ end
+
+
+-local function validate_certificate(cert)
 -  local _, err = openssl_x509.new(cert)
 -  if err then
 -    return nil, "invalid certificate: " .. err
 -  end
 -
 -  return true
-+  return nil, "openssl: not yet implemented"
- end
- 
- 
- local function validate_key(key)
+-end
+-
+-
+-local function validate_key(key)
 -  local _, err =  openssl_pkey.new(key)
 -  if err then
 -    return nil, "invalid key: " .. err
 -  end
 -
 -  return true
-+  return nil, "openssl: not yet implemented"
- end
- 
- 
+-end
+-
+-
+ local typedefs = {}
+
+
+@@ -385,13 +364,13 @@ typedefs.sni = Schema.define {
+
+ typedefs.certificate = Schema.define {
+   type = "string",
+-  custom_validator = validate_certificate,
++  custom_validator = x509.validate_certificate,
```

Comment:
[At the top of this file ](https://github.com/Kong/goks/blob/7ea3fa09039dc76fd531907a8531150803207f10/patches/lua-tree.patch#L49) there is a summary of the changes included in the patch. Can you please replace the entry about removing `openssl` with a description of the new additions?

Reply:
done